randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -668,16 +672,32 @@ Extra - asset_id: 225636855
               Any of the following:
                   Space Jump Boots
                   Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
-          Space Jump Boots and Slope Jump (Intermediate)
-          No Gravity Suit and Slope Jump (Advanced) and Enabled Allow Dangerous Gravity Suit Logic
+          All of the following:
+              # https://youtu.be/B56Q854PHwc
+              Space Jump Boots and Slope Jump (Intermediate) and Gravityless Underwater Movement (Beginner)
+          No Gravity Suit and Slope Jump (Advanced) and Gravityless Underwater Movement (Intermediate) and Enabled Allow Dangerous Gravity Suit Logic
   > Pickup (Wavebuster)
       Any of the following:
           All of the following:
               Missile ≥ 36
               Any of the following:
                   Space Jump Boots
                   Morph Ball Bomb and Morph Ball and Bomb Jump (Expert)
-          Space Jump Boots and Combat/Scan Dash (Beginner) and Standable Terrain (Intermediate)
+          All of the following:
+              Space Jump Boots
+              Any of the following:
+                  All of the following:
+                      # https://youtu.be/2jyvwogPSDI
+                      Combat/Scan Dash (Advanced) and Standable Terrain (Beginner)
```

Comment:
Combat/Scan Dash (Advanced) -> Combat/Scan Dash (Intermediate)
Agree it is more difficult than Beginner, but is not as difficult as other Advanced category dashes.

Reply:
I think you are correct on this, I've made the change.